### ISSUE TITLE
GUI wallet design request.

### ISSUE THREAD

**Issue by CaptainSkywave** [wagerr/wagerr#62]
GUI wallet.
When on Masternodes tab - selecting (clicking on) a MN, highlights the line with red colour fill.
If I click somewhere outside the MN list (inside the wallet) - the highlighted red is still present.
One can click another node, but then again, that line will remain highlighted.
Same happening on the Transactions tab.

I find this somewhat annoying and would like to be able to see a clean list w/o highlights - when I am done with my pinpointing.

**Comment by WagerrTor** [comment]
@CaptainSkywave 
do you mean something like adding mouse event tracking when your mouse is out of the frame that red is greyed out or not marked at all?

And do you mean that when you move your mouse back into the table, that it follows your mouse by highlighting the row over which the mouse goes?

Currently it is singleselection, but we could make it ability for multiselection.

**Comment by CaptainSkywave** [comment]
When I have selected (leftclicked) on a node or a transaction, then the line stays highlighted (filled in red) and when clicking somewhere else in a white/empty area I would expect the red fill of that line should go away, but it doesn't - the red fill stays there.
I can select/click another line - then that line is filled with red - and the red fill goes away from the previous selected line.
I just want all red fill to go away when clicking outside in an empty white area - or when I select another tab to view - and when I then go back to the previous tab, where I had selected a line that was red filled - I would like the fill to have disappeared, until I select a line again.
